packages/fxa-settings/src/pages/CompleteResetPassword/en.ftl

```diff
@@ -0,0 +1,25 @@
+## CompleteResetPassword component
+
+# The user followed a password reset link, but that link is expired and no longer valid
+complete-reset-pw-link-expired-header = Reset password link expired
```

Comment:
It looks like we may have some heavy overlap with https://github.com/mozilla/fxa/pull/14699 -- what are thoughts about refactoring these views (possibly in a separate PR?) to share a component?

Reply:
Oh wow, these are REALLY similar. Would definitely be best to share component(s) for these two (and possibly more pages?). Also wondering if these should be broken up into 2-3 components: link expired/link damaged/link valid?

Reply:
Or even to use those IDs.

Also @vbudhram Do we want to call this page `ResetPasswordComplete` to match the other reset PW pages? Would nicely group them together in the tree. I see we have `ConfirmResetPassword` too though, I wonder if that should be `ResetPasswordConfirm` 🤔 I get wanting to stick with what's in the URL too though, I can just see this all getting kind of messy under `pages/` when we've added a bunch more pages.